processor/src/decorators/mod.rs

```diff
@@ -296,17 +299,17 @@ mod tests {
         ];
 
         let stack_inputs = StackInputs::try_from_values(stack_inputs).unwrap();
-        let advice_inputs = AdviceInputs::default().with_merkle_sets(vec![tree.clone()]).unwrap();
+        let advice_inputs = AdviceInputs::default().with_merkle_store(store);
         let advice_provider = MemAdviceProvider::from(advice_inputs);
         let mut process = Process::new(Kernel::default(), stack_inputs, advice_provider);
         process.execute_op(Operation::Noop).unwrap();
 
-        // inject the node into the advice tape
+        // inject the node into the advice stack
         process
             .execute_decorator(&Decorator::Advice(AdviceInjector::MerkleNode))
             .unwrap();
 
-        // read the node from the tape onto the stack
+        // read the node from the stack onto the stack
```

Comment:
Let's change this to: "pop the node from the advice stack and push it onto the operand stack"

Reply:
Done!